routers/private/push_update.go

```diff
@@ -51,10 +43,5 @@ func PushUpdate(ctx *macaron.Context) {
 		}
 		return
 	}
-
-	log.Trace("TriggerTask '%s/%s' by %s", repo.Name, branch, pusher.Name)
-
-	go models.HookQueue.Add(repo.ID)
```

Comment:
This line is actually removed (as opposed to simply being moved to `models/update.go`), since the repo ID is already added to `HookQueue` by `CommitRepoAction(..)`